include/eve/module/core/regular/impl/reduce.hpp

```diff
@@ -48,8 +48,12 @@ template<scalar_value T, typename N, typename Callable>
 EVE_FORCEINLINE auto
 reduce_(EVE_SUPPORTS(cpu_), splat_type const& s, wide<T, N> v, Callable f) noexcept
 {
-  auto op = find_reduction(f, s);
-  return op(s, v);
+  if constexpr (std::same_as<Callable, callable_add_>) return eve::detail::sum[splat](v);
```

Comment:
find reduction can return sum.
I don't understand this change.

Reply:
This is just temporary until all reductions callables are reworked.
`find_reduction` returns an object which is expected to be called with a `splat_type` parameter. We cannot do that anymore as it has been changed to a callable option.